var/spack/repos/builtin/packages/spath/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Spath(CMakePackage):
+    """Represent and manipulate file system paths"""
+
+    homepage = "https://github.com/ECP-VeloC/spath"
+    git      = "https://github.com/ecp-veloc/spath.git"
```

Comment:
I noticed there is a difference in capitalization between git and homepage (same for other packages). We should at least be consistent here.